exercise-src/react-native/11-navigation/03-problem/src/screens/RestaurantDetails/RestaurantDetails.tsx

```diff
@@ -36,7 +36,7 @@ const RestaurantDetails: React.FC = () => {
     <Screen>
       <RestaurantHeader restaurant={restaurant} />
       <Button onPress={() => console.warn("Place an order")}>
-        Place My Order!
+        Place an order
```

Comment:
I made the buttons say "Place My Order" because, if this were an actual product called PMO, the marketing team would absolutely put that on the button. Mainly just for lol.

Reply:
Hope you saw this in the description:

> Use “Place an order” in the Restaurant Details view because it’s the button goes to the Order page (where there’s a “Place My Order!” Button).

This button goes to the Order page, and the “actual” order button still has the whimsy you mention.